contracts/Killable.sol

```diff
@@ -0,0 +1,42 @@
+pragma solidity 0.5.10;
+
+import "./SafeMath.sol";
+import "./Owned.sol";
+
+contract Killable is Owned {
+    using SafeMath for uint;
+    
```

Comment:
You don't need delay, if something bad happened and the owner needs to kill the contract; they should be able to do it right away.